drift/info_parser.py

```diff
@@ -189,6 +203,23 @@ def _create_comparison(systems, info_name):
     }
 
 
+def _is_unique_rec_name(info_name):
+    """
+    helper method to see if we should use the uniqueness recommendation on the
+    fact comparison
+    """
+    if not info_name.startswith("network_interfaces"):
+        return
+    elif info_name.startswith("network_interfaces.lo."):
+        return
+    elif info_name.endswith(".ipv4_addresses"):
+        return True
+    elif info_name.endswith(".ipv6_addresses"):
+        return True
+    elif info_name.endswith(".mac_address"):
+        return True
```

Comment:
We need the fqdn included as well as a unique recommendation

Reply:
Thoughts on a dictionary stored either in a separate python file or json/yaml object that maps property to its operator? And default to 'same' if no entry.

Reply:
I can split it out into an array which may help with readability